Sample Examples/Testing2.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sample_Examples
+{
+    internal class Testing2
+    {
```

Comment:
Please check this method is empty. Implement this method body as per requirement

Reply:
Resolved PR Comments and pushed changes to PR